playback/src/lib.rs

```diff
@@ -1,31 +1,10 @@
+#![deny(rust_2018_idioms)]
+
 #[macro_use]
 extern crate log;
 
-extern crate byteorder;
-extern crate futures;
-extern crate shell_words;
-
-#[cfg(feature = "alsa-backend")]
```

Comment:
I'm probably missing something here, but where are the libraries for features included conditionally at compile time imported now that this has been removed?

Reply:
In Rust 2018, Cargo takes care of linking the libraries in and also imports the crate name into the namespace, so all `extern crate` lines are redundant (except for `#[macro_use]` ones which still have a unique behavior).